src/components/modals.js

```diff
@@ -41,6 +41,10 @@ class ContentTable extends Component {
           <div className="modal-title-container">
             <div className="modal-title">
               <h3> {this.currentICO.name || this.currentICO.information.aliasName}</h3>
+              <p className="ico-paragraph">
+                These criteria are chosen based on the concept of trustless-trust. You can read more about it &nbsp;
+                <a href="https://docs.google.com/spreadsheets/d/1nHnl1vvDRQ5wsVXrGbkMAvfyw1D9LfWoP1q4fa0Ld7U/edit#gid=0" target="_blank">here</a>&nbsp;If you want to submit new criteria please do it via github&nbsp;<a href="https://github.com/Neufund/ico-transparency-monitor#adding-custom-icos-to-the-transparency-monitor" target="_black">here</a>.
```

Comment:
This line is long I'm pretty sure it won't pass eslint. We don't need to fix all of eslint errors now but at least lets try not to introduce new ones.